### PR TITLE
Tailscale Reenable Service after Update If it was Enabled Previously

### PR DESCRIPTION
**Pull request by dyehya**
This will hopefully check if the tailscale service is enabled and if it is reenable it after the update. This does not currently try to check if tailscale is up and make sure it is up after running. There are some issues with trying to run tailscale status even with the --json flag which is subject to change so I did not attempt anything more robust than this.

This still needs to be tested but will hopefully allow pacman -Syu to run and update tailscale without resulting in network connectivity loss after a reboot. I included the older commit with the tailscale up after the update to "force" it to be available but didn't trust that change as if someone has the service enabled but not currently up, they may not want it to put itself up after an update.

### CODE REVIEW

File: packages/tailscale-pikvm/tailscale-pikvm.install [hunk 1]
```diff
@@ -1,8 +1,15 @@
+pre_upgrade() {
+	service_enabled=$(systemctl is-enabled tailscaled)
```
Comment (mdevaev) [inline packages/tailscale-pikvm/tailscale-pikvm.install]: make it like this: `$(systemctl is-enabled tailscaled || true)`

File: packages/tailscale-pikvm/tailscale-pikvm.install [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+
 post_install() {
 	post_upgrade
 }
 
 post_upgrade() {
 	mkdir -p /var/lib/tailscale || true
 	chmod 700 /var/lib/tailscale || true
+	if [ "$service_enabled" == "enabled" ]; then
+		systemctl enable --now tailscaled
```
Comment (mdevaev) [inline packages/tailscale-pikvm/tailscale-pikvm.install]: remove --now, change this line like `systemctl enable tailscaled || true`

Comment (dyehya) [reply packages/tailscale-pikvm/tailscale-pikvm.install]: Updated

File: packages/tailscale-pikvm/tailscale-pikvm.install (hunk 1 above)
Comment (dyehya) [reply packages/tailscale-pikvm/tailscale-pikvm.install]: @mdevaev updated

**Pull request merged**